src/main/java/com/google/devtools/build/lib/runtime/LocalInstrumentationOutput.java

```diff
@@ -17,23 +17,32 @@
 
 import com.google.devtools.build.lib.buildtool.BuildResult.BuildToolLogCollection;
 import com.google.devtools.build.lib.vfs.Path;
+import com.google.devtools.build.lib.vfs.PathFragment;
 import com.google.errorprone.annotations.CanIgnoreReturnValue;
 import java.io.IOException;
 import java.io.OutputStream;
+import javax.annotation.Nullable;
 
 /** Used when instrumentation output is written to a local file. */
 final class LocalInstrumentationOutput implements InstrumentationOutput {
   private final Path path;
   private final String name;
+  @Nullable private final String convenienceName;
 
-  LocalInstrumentationOutput(String name, Path path) {
+  LocalInstrumentationOutput(String name, Path path, String convenienceName) {
     this.name = name;
     this.path = path;
+    this.convenienceName = convenienceName;
   }
 
   @Override
-  public void publish(BuildToolLogCollection buildToolLogCollection) {
+  public void publish(BuildToolLogCollection buildToolLogCollection) throws IOException {
     buildToolLogCollection.addLocalFile(name, path);
+    if (convenienceName != null) {
+      var link = path.getParentDirectory().getChild(convenienceName);
+      link.delete();
+      link.createSymbolicLink(PathFragment.create(path.getBaseName()));
```

Comment:
This code path doesn't seem to be executed with `bazel info --generate_json_trace_profile`.

Reply:
Thanks; I moved the symlinking logic into `BlazeCommandDispatcher.execExclusively`.